elasticpath/react-pwa-reference-storefront
# Rs 663

Description:


Builds a general component `PaymentSelectorMain` that renders payment options for all versions of EP in both the profile and orders resource.  Check out the component README for more details: `src/components/PaymentSelectorMain/README.md`.

Deletes `ProfilePaymentMethods` in place of the aforementioned component.

The `CheckoutPage.tsx` and `ProfilePage.tsx` both integrate this component.

NOTE:  Ability to select this component will be added in future PR's to accomodate for PO number entry designs.

Linting:


Component Updates:


Tests:


1. connect to `https://reference.elasticpath.com` and go through checkout flow and payment options in profile
2. connect to the latest payments release and do the same.  Notice the application functions for both.
3. Checkout storybook stories.  
4. Read through storybook documentation.

Documentation:
